x-pack/plugins/observability_solution/apm/public/components/app/entities/logs/logs_service_overview.tsx

```diff
@@ -52,7 +66,52 @@ export function LogsServiceOverview() {
       end={end}
     >
       <ChartPointerEventContextProvider>
-        <AddAPMCallOut />
+        {isLogsApmCalloutEnabled ? (
+          <>
+            <AddAPMCallOut
+              onClose={() => {
+                setIsLogsApmCalloutEnabled(false);
+              }}
+            />
+            <EuiSpacer size="l" />
+          </>
+        ) : null}
+        <EuiCallOut
```

Comment:
Do we always want to display the callout?

Reply:
good catch

Reply:
done https://github.com/elastic/kibana/pull/189644/commits/2954e566e1241158549f3504e3e09f8efdcaf6b3